docs/02. Quick Start.md

```diff
@@ -93,10 +93,14 @@ the following code in the `onBootstrap` method of your `Module.php` class:
 ```php
 public function onBootstrap(EventInterface $e)
```

Comment:
Change typehint to MvcEvent (since there's no `getApplication()` method in `EventInterface`)